forge/db/migrations/20220808-02-create-default-team-types.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Add default TeamTypes to table and ensure all teams are set to starter type.
+ *
+ * This migration is also called whenever FlowForge starts in order to insert
+ * the default set of TeamTypes. It only applies them if there are no types found
+ * in the table.
+ */
+
+module.exports = {
+    up: async (context) => {
+        const count = await context.sequelize.query('select count(id) as count from `TeamTypes`', { type: context.sequelize.QueryTypes.SELECT })
+        if (count[0].count === 0) {
+            await context.bulkInsert('TeamTypes', [
+                {
+                    name: 'starter',
+                    enabled: true,
+                    description: `Manage your personal projects
+ - Up to 3 team members
+ - More features to highlight
+`,
+                    properties: JSON.stringify({
+                        userLimit: 0
+                    }),
+                    createdAt: new Date(),
+                    updatedAt: new Date()
+                }
```

Comment:
Need to get consensus on these values for the default 'free' team before this is merged.